spec/fixtures/responses/mutations/createFreak/default.json

```diff
@@ -0,0 +1,10 @@
+{
+  "status": 200,
+  "body": {
+    "errors": [
+      {
+        "message": "No query string was present"
+      }
```

Comment:
It's weird to expect an error in the response in this case, rather we want the created Freak.